control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```
@@ -207,6 +211,20 @@ func NewKubeAPIServerParams(hcp *hyperv1.HostedControlPlane, images map[string]s
 	}
 	params.KubeConfigRef = hcp.Spec.KubeConfig
 	params.OwnerRef = config.OwnerRefFrom(hcp)
+
+	cfgs := []client.Object{
+		&params.APIServer,
+		&params.Authentication,
+		&params.FeatureGate,
+		&params.Network,
+		&params.OAuth,
+		&params.Image,
+		&params.Scheduler,
+	}
+	err := config.ExtractConfigs(hcp, cfgs)
+	if err != nil {
+		log.Error(err, "Errors encountered extracting cluster configs")
```

isn't this swallowing the error?  how will invalid config propagate?